policy/modules/apps/gpg.if

```diff
@@ -341,6 +432,24 @@ interface(`gpg_runtime_filetrans',`
 	userdom_search_user_runtime($1)
 ')
 
+########################################
+## <summary>
+##	Do not audit attempt to getattr gpg runtime dirs.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed access.
```

Comment:
This interface is not allowing access.

Reply:
It simply prevents auditing. It does not grant or deny any permimssion.

Reply:
I suppose you mean the comment... I'll amend that.